### PR TITLE
chore: fix eslint issues

### PR DESCRIPTION
**Pull request by belom88**
- Enable eslint for the tile-converter module;
- Fix all lint issues;

### CODE REVIEW

File: modules/tiles/src/tileset/tileset-3d.ts [hunk 1]
```diff
@@ -841,7 +841,7 @@ export class Tileset3D {
           break;
         default:
       }
-    } else if (this.type === TILESET_TYPE.TILES3D) {
+    } else if (this.type === 'TILES3D') {
```
Comment (mspivak-actionengine) [inline modules/tiles/src/tileset/tileset-3d.ts:844]: What is the reason for this change? TILESET_TYPE enum is still being used here, e.g. line 809.
If it's decided not use enums (don't know why), it should be done across all the code, I guess.

Comment (belom88) [reply modules/tiles/src/tileset/tileset-3d.ts:844]: eslint warning/error was here because`string` compared with `enum`. I guess it is considered as a potential issue

**Review by mspivak-actionengine — APPROVED** [review]

**Review by ibgreen — APPROVED** [review]
